test/common/test_runner.go

```diff
@@ -115,6 +118,58 @@ func TearDown(client *Client) {
 	}
 }
 
+// CopySecret will copy a secret from one namespace into another.
+// If a ServiceAccount name is provided then it'll add it as a PullSecret to
+// it.
+// It'll either return a pointer to the new Secret or and error indicating
+// why it couldn't do it.
+func CopySecret(client *Client, srcNS string, srcSecretName string, tgtNS string, svcAccount string) (*corev1.Secret, error) {
```

Comment:
The function is inconsistent. Sometimes it calls `Fatalf` sometimes returns errors (should always do second, though).

Reply:
yep missed that - fixed